src/test/loader.ts

```diff
@@ -191,7 +202,7 @@ export class Loader {
       name: takeFirst(this._configOverrides.name, projectConfig.name, this._config.name, ''),
       testDir,
       testIgnore: takeFirst(this._configOverrides.testIgnore, projectConfig.testIgnore, this._config.testIgnore, []),
-      testMatch: takeFirst(this._configOverrides.testMatch, projectConfig.testMatch, this._config.testMatch, '**/?(*.)+(spec|test).[jt]s'),
+      testMatch: takeFirst(this._configOverrides.testMatch, projectConfig.testMatch, this._config.testMatch, '**/?(*.)+(spec|test).+(ts|js|mjs)'),
```

Comment:
Is the + there intended? This would mean:

`foo.spec.tststs` would also be a match.

Reply:
I'll fix the +(spec|test) in a follow up.